src/block_cache/block_chunkcache.c

```diff
@@ -261,6 +261,37 @@ __chunkcache_eviction_thread(void *arg)
  * __wt_chunkcache_get --
  *     If the cache has the data at the given size and offset, copy it into the supplied buffer.
  *     Otherwise, read and cache the chunks containing the requested data.
+ *
+ * The locks protect two data structures: (1) the buckets of the hash table that keeps track of the
+ *     allocated chunks and the (2) LRU list of chunks, used by eviction. Each bucket in the hash
+ *     table is protected by a single spinlock. The LRU list is also protected by a single lock.
+ *
+ * Lock ordering: * If we ever need to hold a bucket lock and the LRU list lock simultaneously, we
```

Comment:
Would it make sense to move this down to become the final paragraph? There's already a section on locking protocol there, and "how to hold both locks" is related.